datasets/elkarhizketak/dummy/plain_text/1.0.0/dummy_data/elkarhizketak-dev-v1.0.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
can you delete the `datasets/elkarhizketak/dummy/plain_text/1.0.0/dummy_data/` directory and all its content ? We just need the `dummy_data.zip` file

Reply:
Done!